features/step_definitions/basic_steps.rb

```diff
@@ -568,3 +568,11 @@
   expect(stderr).not_to include "Error"
   expect(status).to be_success
 end
+
+Then(/^I should have a page with a title: "([^"]*)"$/) do |title|
+  expect(page).to have_title 'Harrow Community Network'
```

Comment:
@lollypop27 you're hard coding the title's value in this test instead of using the one you extracted in `title`